Binge/BingePlayerViewController.swift

```diff
@@ -19,6 +19,8 @@ public enum LayoutState: String {
     case pictureInPicture
 }
 
+public var initializedPlayers: [AVPlayer] = []
```

Comment:
Here, you are introducing some global state, which should be avoided at all costs - especially in frameworks.
Additionally, a reference to each instantiated player is stored in the array and never removed. In this way, the required memory is never freed.

I found this [guide by Apple](https://developer.apple.com/documentation/avfoundation/avaudiosession/responding_to_audio_session_interruptions), which describes the standardized way for our issue. Please have a look at it - especially `interruptionNotification` and/or `silenceSecondaryAudioHintNotification`.